AckooSDK/Classes/AckooSDKManager1.swift

```diff
@@ -0,0 +1,171 @@
+//
+//  AckooSDKManager1.swift
+//  AckooSDK
+//
+//  Created by Sally Ahmed1 on 12/1/20.
+//
+
+import Foundation
+import UIKit
+
+@objc public protocol AckooSDKType1 {
+    func initSession()
+    func continueActivity(userActivity: NSUserActivity)
+    func identify(id: String, profile: [String: Any], callback: @escaping (_ succeeded: Bool, _ response: Any) -> Void)
```

Comment:
is AckooError not going to be returned ?

Reply:
i save the signature of the old function, but I think we should change it